src/app/features/issue/providers/azuredevops/azuredevops-issue/azuredevops-issue.model.ts

```diff
@@ -0,0 +1,18 @@
+export type AzuredevopsIssueReduced = Readonly<{
+  id: number;
+  url: string;
+  title: string;
+  state: string;
+  updated_at: string;
+}>;
+
+export type AzuredevopsIssue = AzuredevopsIssueReduced &
```

Comment:
I am unsure where it is used, but is not àssigned to`missing? That may be relevant for filtering, e.g. you only want to see all issues assigned to yourself?

(Also BTW; depending on your ADS template AFAIK it is not called issue but actually [backlog item](https://learn.microsoft.com/en-us/azure/devops/boards/work-items/about-work-items?view=azure-devops&tabs=scrum-process#work-item-types-wits) AFAIK (which can have different types).

Speaking about types, is not that missing, too? E.g. I may only want to filter tasks in one use case. In another one, I may however want to filter tasks an product backlog items.